src/robotide/context/coreplugins.py

```diff
@@ -23,7 +22,9 @@ def get_core_plugins():
     from robotide.log import LogPlugin
     from robotide.searchtests.searchtests import TestSearchPlugin
     from robotide.spec.specimporter import SpecImporterPlugin
+    from robotide.recentfiles import RecentFilesPlugin
 
-    return [RunAnything, RecentFilesPlugin, PreviewPlugin, SpecImporterPlugin,
-            EditorPlugin, TextEditorPlugin, KeywordSearch, LogPlugin, TestSearchPlugin]
+    # removed RecentFilesPlugin,
+    return [RunAnything, PreviewPlugin, SpecImporterPlugin,
+            EditorPlugin, TextEditorPlugin, KeywordSearch, LogPlugin, TestSearchPlugin, RecentFilesPlugin]
 
```

Comment:
Why is the order of plugins just changed? Maybe revert all the changes to this file.